app/javascript/components/public_health/PatientsTable.js

```diff
@@ -56,12 +57,14 @@ class PatientsTable extends React.Component {
           { field: 'symptom_onset', label: 'Symptom Onset', isSortable: true, tooltip: null, filter: this.formatSymptomOnset },
           { field: 'risk_level', label: 'Risk Level', isSortable: true, tooltip: null },
           { field: 'monitoring_plan', label: 'Monitoring Plan', isSortable: true, tooltip: null },
+          { field: 'reporter', label: 'Reporter ID', isSortable: true, tooltip: null },
```

Comment:
would it be more useful to make this a link instead of just plain text?

Reply:
Asked in [slack](https://mitrecorp.slack.com/archives/C01A6P94R9B/p1623771981017400)

Reply:
Made it a link https://github.com/SaraAlert/SaraAlert/pull/937/commits/e09bee26d561faa49c6acc0a1f05650898fa4047